### PR TITLE
Display list of items on the homepage

### PR DESCRIPTION
**Pull request by NduatiKagiri**
In this branch I:

- Designed the homepage
- Styled the homepage
- Added function to fetch data from the API and store it on localStorage
- Added function to fetch data from localStorage and display it on the homepage

### CODE REVIEW

File: src/modules/getShows.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const myURL = 'https://api.tvmaze.com/shows';
+
+const getShows = async () => {
+  const response = await fetch(`${myURL}`);
+  let datas = await response.json();
+  datas = datas.slice(0, 6);
```
Comment (dqarias) [inline src/modules/getShows.js]: [OPTIONAL] Kindly remove this line, it's better to show all the elements and no limit them.

File: src/styles/homepage.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.shows {
+  width: 100%;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 45px 30px;
+  padding: 20px 0;
+  background: #242222;
```
Comment (dqarias) [inline src/styles/homepage.css]: [OPTIONAL] Kindly you can add more columns

File: src/styles/homepage.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.show {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
```
Comment (dqarias) [inline src/styles/homepage.css:14]: [OPTIONAL] Kindly you can put the name and likes in the margin of each image container.

File: src/modules/getShows.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      <img class="show-img" src="${show.image}" alt="">
+      <div class="name-likes">
+        <h6>${show.name}</h6>
+        <div class="likes">
+          <img id="like${show.id}" src="https://nduatikagiri.co.ke/tvshows/love-red-48.png" alt="">
+          <p><strong>5</strong> likes</p>
+        </div>
+      </div>
+      <button type="button" class="homebtnc" id="comments${show.id}">Comments</button>
+      <button type="button" class="homebtnr" id="reservations${show.id}">Reservations</button>
```
Comment (dqarias) [inline src/modules/getShows.js]: [OPTIONAL] You should remove the reservation button, is no necessary for our project

**Review by dqarias — APPROVED** [review]
## Hi @NduatiKagiri 👋,
Good job so far!, This milestone is complete! 💪

There is nothing else to say other than... it's time to merge it :shipit:

## To Highlight 🎉
No linter errors. ✔️
Gitflow is followed. ✔️
HTML & CSS best practices followed. ✔️

## Suggestions ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

File: src/modules/getShows.js (hunk 1 above)
Comment (NduatiKagiri) [reply src/modules/getShows.js]: They are too many and we will need Pagination to show all of them. Please let us display around 24 of them

File: src/styles/homepage.css (hunk 2 above)
Comment (NduatiKagiri) [reply src/styles/homepage.css]: Just like you had suggested earlier, kindly lets do the styling after we add all the functions

Comment (dqarias) [reply src/styles/homepage.css]: you are right, so let's do it later

File: src/modules/getShows.js (hunk 1 above)
Comment (dqarias) [reply src/modules/getShows.js]: Ok, let's display only 24 elements

**Pull request merged**